Octokit/Models/Response/Repository.cs

```diff
@@ -58,7 +58,9 @@ public Repository(string url, string htmlUrl, string cloneUrl, string gitUrl, st
             AllowSquashMerge = allowSquashMerge;
             AllowMergeCommit = allowMergeCommit;
             Archived = archived;
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
I like this approach much better than throwing warns.. this feels like a [ubiquitous language](https://martinfowler.com/bliki/UbiquitousLanguage.html) kinda thing as well; as in we need a common comment language to communicate things such as obsolete fields, etc... so that we can set up automation or simply search when we decided to clean things up.

Reply:
Definitely. Obsoletes should really be there for people trying to use the codebase we produce. If we're obsoleting something, we should really change it locally so we don't use it any more, before packaging it up etc.

Yes, definitely we need some common language, it could get very messy otherwise!